chcemvediet/apps/inforequests/templates/admin/inforequests/action/delete_confirmation.html

```diff
@@ -1,13 +1,42 @@
 {% extends "admin/delete_confirmation.html" %}
-{% load amend prepend from poleno.amend %}
+{% load i18n %}
+{% load amend prepend before after delete from poleno.amend %}
 
 {% block content %}
   {% amend %}
     {{ block.super }}
+    {% after path="./ul[last()]" %}
+      {% if outbound %}
+        <p>Outbound messages will be deleted:</p>
+        <ul>{{ outbound|unordered_list }}</ul>
+      {% endif %}
+      {% if inbound %}
+        <p>Inbound messages will be marked undecided:</p>
+        <ul>{{ inbound|unordered_list }}</ul>
+      {% endif %}
+    {% endafter %}
+    {% if not object.is_last_action %}
+      {% before path=".//form" %}
+        <p>
+          <b>Warning:</b> The deleted action is not the last action in the branch. Deleting it may
+          cause logical errors in the inforequest history.
+        </p>
+      {% endbefore %}
+    {% endif %}
     {% if object.type == object.TYPES.EXPIRATION or object.type == object.TYPES.APPEAL_EXPIRATION %}
       {% prepend path=".//form" %}
         <p><label><input name="snooze" type="checkbox" /> Extend snooze of previous action by {{ ADMIN_EXTEND_SNOOZE_BY_DAYS }} days from today.</label></p>
       {% endprepend %}
     {% endif %}
+    {% if delete_dependency %}
+      {% before path=".//form" %}
+        <p class="errornote">Delete not allowed.</p>
+        <ul>{{ delete_dependency|unordered_list }}</ul>
+      {% endbefore %}
+      {% delete path=".//form//input[@type='submit']" %}
+      {% after path=".//form//input[@name='post']" %}
```

Comment:
@martinmacko47 Asi by sa nam zislo keby sme si vytvorili `edit` template tag.

Reply:
@viliambalaz Ano, pls sprav template tag na upravu atributov elementov:

```
{% set_attributes path="..." aaa="value" bbb="value" ccc=None ddd=True eee=False  %}
```

Kde:
- `aaa="value"` vytvori atrubut `aaa` s hodnotou `"value"`
- `ccc=None` a `eee=False` dropne atrinuty `ccc` a `eee`
- `ddd=True` vytvori atribut `ddd` bez hodnoty

Ak vytvarany atribut uz na elemente existuje, prepise sa. Ak mazany atribut na elemente neexistuje, nestane sa nic.

Malo by to ist spravit jednoducho analogicky ako ostatne template tagy. Aby template tag bral lubovolne mnozstvo argumentov staci napisat:

```
@register.simple_tag(takes_context=True)
def set_attributes(context, path, **kwargs):
  ...
```

Reply:
PR: https://github.com/martinmacko47/chcemvediet/pull/366

Reply:
:ok: